packages/deep-merge/package.json

```diff
@@ -21,7 +24,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://bitbucket.org/colbenson/eb-deep-merge.git"
+    "url": "git+https://github.com/empathyco/x.git"
```

Comment:
This should be the url to the repo, without `git+ssh` or `git+https` as specify in the [documentation examples](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository).

Reply:
I'll fix this `repository` property in the last PR.